### DISCUSSION TITLE
consul implements the skywalking cluster. Where are the token configuration items?

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11913] [Q&A] [answered]
consul implements the skywalking cluster. Where are the token configuration items? Are the configuration items incomplete?

<img width="1679" alt="企业微信截图_0f06cac0-8631-4609-968a-b3ef45e0446e" src="https://github.com/apache/skywalking/assets/31468018/bd734e8c-76cc-41c0-8dc7-54f19fb417ac">

**Comment by wu-sheng** [comment, marked as answer]
`cluster/consul/aclToken` in the `application.yml` or `SW_CLUSTER_CONSUL_ACLTOKEN`.

All are listed in https://skywalking.apache.org/docs/main/latest/en/setup/backend/configuration-vocabulary/

**Comment by 1450793561** [comment]
For the same set of consul, the cluster mode displays failed to connect

<img width="770" alt="企业微信截图_c785e76d-c035-4f09-be91-9a354932c0b4" src="https://github.com/apache/skywalking/assets/31468018/234c58e7-b7bf-40ec-9b26-290f1d628a3f">



<img width="1534" alt="企业微信截图_76a0d9e8-691b-4a16-8f12-19fac6bc331c" src="https://github.com/apache/skywalking/assets/31468018/ee450cee-3462-4be0-8c30-0a8a16f9442c">
